server/streaming/streaming.go

```diff
@@ -363,6 +364,7 @@ func (s *server) servePortForward(req *restful.Request, resp *restful.Response)
 // criAdapter wraps the Runtime functions to conform to the remotecommand interfaces.
 // The adapter binds the container ID to the container name argument, and the pod sandbox ID to the pod name.
 type criAdapter struct {
+	ctx context.Context
```

Comment:
@haircommander is it ok to store ctx here ?

Reply:
yeah I think we're going to have to. do we create a criAdapter for each new request or for each runtime?